.circleci/ci_secrets.yml

```diff
@@ -1,6 +1,6 @@
 db:
   name: circleci_test
   user: circleci
-  password: ''
+  password: password
```

Comment:
this is because the testing db needs a password? Perhaps we could use `not_secure` so its the same as the local database?

Reply:
Sure. I will do that.

Reply:
Yes, Postgres 12.x docker container in CircleCI requires an explicit password now.